app/src/main/java/org/oppia/android/app/administratorcontrols/AdministratorControlsViewModel.kt

```diff
@@ -64,15 +67,20 @@ class AdministratorControlsViewModel @Inject constructor(
   private fun processAdministratorControlsList(
     deviceSettings: DeviceSettings
   ): List<AdministratorControlsItemViewModel> {
+
     val itemViewModelList: MutableList<AdministratorControlsItemViewModel> = mutableListOf(
-      AdministratorControlsGeneralViewModel()
-    )
-    itemViewModelList.add(
       AdministratorControlsProfileViewModel(
         routeToProfileListListener,
         loadProfileListListener
       )
     )
+
+    if (enableEditAccountsOptionsUi.value) {
+      itemViewModelList.add(
+        AdministratorControlsGeneralViewModel()
```

Comment:
If the `enableEditAccountsOptionsUi` value is true won't the `Edit Accounts` option be visible below `Profiles` because of this current implementation.

Can you please turn the value `true` and share screenshot of how it looks.

Reply:
I used a temporary fix (which is now resolved) - For some reason Android Studio cannot import List properly(I think that is an unresolved problem from Android Studio's IDE side). Because of which I could not use ` val itemViewModelList: MutableList<AdministratorControlsItemViewModel> = emptyList` in order to initialise this variable.

> Can you please turn the value true and share screenshot of how it looks.

Now, after turning the value true, it looks like this: 
![image](https://user-images.githubusercontent.com/64526117/148601532-2c114e42-81da-4417-9d9f-c44bc066b7f8.png)